pytest.ini

```diff
@@ -8,6 +8,8 @@ env =
     D:MONGODB_USERNAME=root
     D:MONGODB_PASSWORD=password123
     D:MONGODB_DATABASE=petisco
+    D:RABBITMQ_MAX_ATTEMPTS_TO_RECONNECT_CONSUMER=3
```

Comment:
I like to have this configuration as a envar. Maybe we should add `PETISCO_` prefix as with another petisco configurations

Reply:
I've just committed an update, I've add a preliminary section in the documentation to gather all the configurations (dummy doc for now)

Reply:
:ok_hand: